integration_tests/test_rough.py

```diff
@@ -5,11 +5,19 @@
         rough_parameters)
 from EMaligner import EMaligner
 from EMaligner.utils import EMalignerException
+from marshmallow import ValidationError
 import json
 import os
+import copy
 
-FILE_ROUGH_TILES = './integration_tests/test_files/rough_input_tiles.json'
-FILE_ROUGH_PMS = './integration_tests/test_files/rough_input_matches.json'
+FILE_ROUGH_TILES = \
```

Comment:
Can these (and the ones in test_montage) use `os.path` or `pathlib` and be relative to `__file__` to help portability?